app/views/provider/admin/backend_apis/mapping_rules/edit.html.slim

```diff
@@ -0,0 +1,5 @@
+= content_for :sublayout_title, 'Edit Mapping Rule'
```

Comment:
Same here, the title is not showing up

![Screen Shot 2019-08-13 at 2 44 50 PM](https://user-images.githubusercontent.com/1842261/62942603-ef7c5800-bdd8-11e9-991c-5d55c36deaf5.png)